frontend/src/components/widgets/FileUploader/UploadFileInfo.ts

```diff
@@ -43,7 +43,9 @@ export type FileStatus = UploadingStatus | UploadedStatus | ErrorStatus
  * This class is immutable because it's used in within FileUploader.state.
  */
 export class UploadFileInfo {
-  public readonly file: File
+  public readonly name: string
+
```

Comment:
nit: Is there supposed to be a space here?

Reply:
Yep. I initially didn't have newlines between these fields, but then the typescript linter got angry at me, so I guess they're just a style convention